src/test/java/com/icthh/xm/tmf/ms/communication/lep/XmCommunicationMsLepProcessingApplicationListenerTest.java

```diff
@@ -38,29 +22,25 @@ public class XmCommunicationMsLepProcessingApplicationListenerTest {
     private JavaMailSender javaMailSender;
 
     @Autowired
-    private XmCommunicationMsLepProcessingApplicationListener listener;
+    private XmCommunicationMsLepContextFactory contextFactory;
 
     @Test
     @SuppressWarnings("unchecked")
     public void testBindExecutionContext() {
 
-        ScopedContext context = new DefaultScopedContext("scope");
-        listener.bindExecutionContext(context);
+        LepContext context = (LepContext) contextFactory.buildLepContext(null);
 
-        assertEquals(4, context.getValues().size());
-        assertNotNull(context.getValue(BINDING_KEY_COMMONS, CommonsExecutor.class));
```

Comment:
pls annotate LepContext by GroovyMap annotation, and just replace getValue to get method, and test have to work without additional changes

this test must work without changes, fail of this test indicate potential fail of existing lep

Reply:
I added a second test